wiki/osu!supporter/es.md

```diff
@@ -8,9 +8,9 @@ tags:
 
 # osu!supporter
 
-**osu!supporter** (o la ***etiqueta osu!supporter***) es un título temporal dado a los jugadores que han apoyado a osu!, de manera voluntaria, una donación monetaria para ayudar a crecer. Los supporter reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el titulo está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en la jugabilidad, desempeño o puntaje y nunca lo hará.
+**osu!supporter** (o el ***tag de osu!supporter***) es un título temporal dado a los jugadores que han apoyado a osu!, de manera voluntaria, con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el titulo está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en la jugabilidad, desempeño o puntaje y nunca lo hará.
```

Comment:
On the page it says "[tag](https://i.imgur.com/fztq8sa.png)" instead of "etiqueta", so I think it would be better if it was the same as the page.

The same if you want to [buy](https://i.imgur.com/sNbRA72.png) one.

Reply:
In the next localisation update it will have that name

Reply:
But I see the photo in Spanish

Reply:
> But I see the photo in Spanish

the names of the items

Reply:
Ah, yes, that part has always been like that. I was referring to before making the purchase; anyway, it's just a suggestion, it doesn't have to be the right one.